src/transformers/processing_utils.py

```diff
@@ -830,6 +827,60 @@ class MyProcessingKwargs(ProcessingKwargs, CommonKwargs, TextKwargs, ImagesKwarg
             output_kwargs[modality].update(output_kwargs["common_kwargs"])
         return output_kwargs
 
+    def _check_reversed_images_text(self, images, text):
```

Comment:
This function might be too problem-specific to be included in the base processor class, but since several VLMs are affected by this issue and it’s only a temporary solution pending deprecation, I thought it might make sense to include it here. It seems more practical than having the function copied across multiple processors, and should also simplify the deprecation process. Of course, If you think there's a better location for this or another approach that would be more suitable, I’m open to suggestions.

Reply:
Hmm, for me seems like it's better places in each processor file with copied from statements. I guess there are max 5 or so models that need swapping if i'm not wrong. For changes in general processor file, @ amyeroberts can say more

Reply:
I just spoke with Amy, and a good middle-ground solution would be to keep this function in `processing_utils`, but outside of any class, so that models that need it would explicitly import it from `processing_utils`. This approach avoids cluttering the base Processor class with a very problem-specific function and limits the diffs when adding and deprecating this behavior.